pages/1.11/release-notes/1.11.5/index.md

```diff
@@ -0,0 +1,139 @@
+- DCOS-40577 - Fix dashboard permissions and node tab with correct ACLs for users (non-super users and remote users).
+- DCOS_OSS-3801 - Add support for Docker parameters in Metronome jobs.
+
+## Marathon
+- COPS-3505/MARATHON-8326 - Add `wipe=true` support for pod's instances endpoint to automatically drain and decomission agent nodes.
+- MARATHON-7568 - Prevent ZK credentials leak in /v2/info.
+- MARATHON-8084 - Clean up Marathon API POST header. 
+- MARATHON-8368 - Fix offer rejection statistics error when maintenance mode is enabled. 
+
+## Mesos
+- COPS-3616/DCOS-39973/DCOS-40162 - Fix container launch failures that occurred due to Mesos-bridge running out of IPs.
+- COPS-3750/DCOS-41224 - Fix for UCR container cleanup `EBUSY` issue, which restarts EdgeLB tasks.
+- DCOS-38225 - Prevent task loss by fixing error handling failures of Mesos-IAM interaction. [enterprise type="inline" size="small" /]
+- DCOS-40410 - Bump Mesos to nightly [1.5.x eacabd7](https://github.com/mesosphere/mesos/blob/eacabd7/CHANGELOG).
+
+## Networking
+- COPS-3279/COPS-3576/DCOS-37703/DCOS-39721 - Fix erroneous values in service addresses stats. Bump dcos-net.
+- COPS-3472/DCOS-38932 - Enable access to Marathon app via overlap network. 
+- COPS-3520/DCOS-39999 - Fix DC/OS OSS build failure that occurred due to segmentation violation.
+- DCOS-38600 - Remove deadlock when SSL sockets are simultaneously sending/receiving data.
+- DCOS-39707 - Fix clustering issues with `etcd`.
+- DCOS_OSS-3697 - Fix connectivity issue between bridge and overlay networks.
+- DCOS_OSS-3841 - Update CNI plugin versions to v0.7.1.
+
+## Platform
+- COPS-3568/DCOS-39883 - Add permissions to `dcos_diagnostics_master` to read marathon state.
+- COPS-3658/DCOS-21258 - Remove `dcos-history-service`from UI. 
+- DCOS-22194 - Fix `dcos-metrics` pkgpanda build to utilize Docker. 
+- DCOS-37454 - Fix Prometheus inconsistent output with stored metrics.
+
+[enterprise]
+## Security
+[/enterprise]
+- COPS-2988 - Provide access permissions with full capabilities to the user. 
+- COPS-3195 - Fix cluster authentication issues running in strict mode and issues arising from transient errors in the `ip-detect` script.
+- COPS-3485/DCOS_OSS-3937 - Prevent loss of ZK myid configuration file by recreating the deleted myid configuration file and restarting exhibitor 
+- DCOS-38655 - Force all login requests through Admin Router. 
+- DCOS-39259 - Add error handling for `StartTLS` in case of socket failure.
+- DCOS-39260 - Fix LDAP login failure after upgrade from 1.11.1 to 1.11.3.
+- DCOS_OSS-3932 - Update Java version from 8u151 to 8u181.
+
+## Storage
+ - DCOS-39891 - Add a new permission to `dcos_diagnostics_master` service account user during bootstrap.
+
+
+# Notable Changes in DC/OS 1.11.5
+
+- Update DC/OS UI for [1.11+v1.19.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.19.0/CHANGELOG.md). [oss type="inline" size="small" /]
```

Comment:
DC/OS UI changes need not be mentioned in notable changes, Anita. (I think it is a mistake that it is mentioned in CHANGELOG.md under Notable changes). You can include them in bug fixes.
And move the Notable changes upto the top of this page.

Reply:
@orsenthil `Notable Changes` section should be placed below `Fixed Issues` section, as per the layout we've discussed in our RN process improvement meeting. Let mw know, if you still think this move is essential.